scala/ppml/demo/ppml-conf.yaml

```diff
@@ -1,5 +1,7 @@
 ##### Server property
 clientNum: 2
+privateKeyFilePath: /ppml/trusted-big-data-ml/work/keys/server.pem
```

Comment:
This path is OK for container. But may cause problem for local run. Need add comments in README.

Reply:
Updated in README.